pkgs/development/python-modules/mathutils/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "mathutils";
+  version = "3.3.0";
+  format = "setuptools";
+
+  src = fetchPypi {
```

Comment:
fetching from github ( or original repo ) is prefered since packages published to pypi usually don't have tests

Reply:
In this case, this project doesn't have tests, and the maintainer doesn't tag which commits they released. That's why I decided to fetch from pypi.

But after thinking about this, I think it's better to fetch from gitlab indeed, also because it makes easier to audit the code. After all, I have no guarantee the pypi package is shipping the code in the repository... So changing that, yes!